### PR TITLE
Enhamcing the multipath_test.py with new scenarios.

### PR DESCRIPTION
**Pull request by Naresh-ibm**
Enhanced the script with few more scenarios on failing and
re-instating the multipaths.

Signed-off-by: Naresh Bannoth <nbannoth@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
this patch is with respective of #596

**Comment by narasimhan-v** [comment]
Commit message `s/Enhamcing/Enhancing/`

File: io/disk/multipath_test.py [hunk 1]
```diff
@@ -103,13 +102,15 @@ def test(self):
...
             # Path Selector policy
+            self.log.info("changing Selector policy")
             for policy in ["service-time", "round-robin", "queue-length"]:
                 cmd = "path_selector \"%s 0\"" % policy
                 multipath.form_conf_mpath_file(defaults_extra=cmd)
                 if multipath.get_policy(path_dic["wwid"]) != policy:
                     msg += "%s for %s fails\n" % (policy, path_dic["wwid"])
 
             # Blacklist wwid
+            self.log.info("Block listing WWIDs")
```
Comment (narasimhan-v) [inline io/disk/multipath_test.py]: Blacklisting

File: io/disk/multipath_test.py [hunk 2]
```diff
@@ -125,6 +126,64 @@ def test(self):
...
+            self.log.info(" Failing and reinstating the individual paths")
+            for path in path_dic["paths"]:
+                if multipath.fail_path(path) is False:
+                    msg += "test failed while failing %s" % path
+                elif multipath.reinstate_path(path) is False:
+                    msg += "test failed while reinstating %s" % path
+        self.mpath_svc.restart()
+        # Failing n-1 paths for short time and reinstating back
+        self.log.info("Failing and reinstating the n-1 paths")
+        for wwid in self.wwids:
```
Comment (narasimhan-v) [inline io/disk/multipath_test.py]: lines 140 to 144 are redundant. Check the `setUp()` and re-use them

Comment (narasimhan-v) [reply io/disk/multipath_test.py]: And other usages of this code too

**Comment by narasimhan-v** [comment]
@Naresh-ibm Unless some comments are clarified, there is no point in sending a new PR with same old code.

Also, please close the old PR when raising a V2 of it.

**Comment by Naresh-ibm** [comment]
added the suggest changes and created a new level of PR #672 , hence closing it

**Pull request closed without merging**